src/layer/__tests__/cache.ts

```diff
@@ -1,14 +1,14 @@
-import 'jest-setup';
+import '../../../jest-setup';
 import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 import makeServiceWorkerEnv from 'service-worker-mock';
 import fetch from 'node-fetch';
 
 import { constructFixtureFindTiles } from './fixtures.findTiles';
 import { constructFixtureGetMap } from './fixtures.getMap';
-import { ApiType } from 'src';
-import { setAuthToken } from 'src/auth';
-import { invalidateCaches, CacheTarget } from 'src/utils/Cache';
+import { ApiType } from '../../index';
+import { setAuthToken } from '../../auth';
+import { invalidateCaches, CacheTarget } from '../../utils/Cache';
```

Comment:
Since `setAuthToken`, `invalidateCaches` and `CacheTarget` are exported in src/index.ts, could they be in the import above?

Reply:
Changed

Reply:
Resolving